### ISSUE TITLE
Create Product List Item Component

### ISSUE THREAD

**Issue by piotrwicijowski** [fromITzerotoIThero/IBOS#12]
As a user, I want the landing page to contain list of promoted items displayed. For that I need a reusable Product List Item component to be created.

Technical implementation details:
- Create new component
- Add parameters for image, product name, current price, original price, discount percentage, isDiscounted flag
- In case the isDiscounted flag is not set, display just the original price
- If the isDiscounted flag is set, show the original price, current price, and a small angled strip with the discount percentage over the image
- Make sure that the name of the product does not overflow, and in case it is too long, clamp it at two lines and display ellipsis ... at the end.
- When hovering over the title, display a tooltip over the title with the full product name (in case it is too long and the user wants to see the full text)
- Display the product picture. Make sure that in case the original image is not originally square, the display would be cropped to a square and not squished/stretched

Designs can be found here:
https://design.penpot.app/#/view/0287358f-ba0f-80b3-8001-af163e079ce3?page-id=aa6c056f-666a-8012-8001-dfd5bc1f9f51&section=interactions&index=1

**Comment by fromITzerotoIThero** [comment]
W., you need to help me out! :) 

This one is very veeeery important to me... I can't make my boys display and I really don't know why, what's wrong with the `imgSrc` or rather the path?

I've even added one image to IBOS to test what's wrong but I have no idea what's going on :(

**Comment by fromITzerotoIThero** [comment]
I did manage to insert once I've imported them but there must be some other way...

**Comment by fromITzerotoIThero** [comment]
Also, I'm trying to figure out how to make my `discount_band` div won't appear on top of the image. For some reason the CSS class is not applied to it...

**Comment by fromITzerotoIThero** [comment]
Maaaan, I did like a triple typo check and finally I see what's wrong XP `className={style.discount_band}` instead of `className={styleS.discount_band}`...

**Comment by fromITzerotoIThero** [comment]
Thx! :)

**Comment by fromITzerotoIThero** [comment]
W., as for the tooltip, I'm not sure I understand - it's supposed to display conditionally, only when the product name is truncated?

**Comment by piotrwicijowski** [comment]
No, the tooltip should be always available. The wording that I used may have been confusing - I wrote the "in case..." not as a conditional, but as an explanation for why we may need it.

**Comment by fromITzerotoIThero** [comment]
What a relief! XD Thx!
